pkgs/applications/networking/p2p/tremotesf/default.nix

```diff
@@ -0,0 +1,50 @@
+{ lib
+, cmake
+, pkg-config
+, mkDerivation
+, fetchurl
+, qtbase
+, qttools
+, kwidgetsaddons
+, kwindowsystem
+, fmt
+, libpsl
+, cxxopts
+, wrapQtAppsHook
+}:
+
+mkDerivation rec {
+  pname = "tremotesf";
+  version = "2.3.0";
+
+  doCheck = true;
+
+  src = fetchurl {
+    url = "https://github.com/equeim/tremotesf2/releases/download/${version}/${pname}-${version}.tar.gz";
+    sha256 = "sha256-/zRKPMdik4rDIoKL97zjJTfZ8iHmHTLXcDsWyjPBcJ8=";
+  };
```

Comment:
You should use the [fetchFromGitHub](https://nixos.org/manual/nixpkgs/stable/#fetchfromgithub) wrapper here.

Reply:
Done